analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.Loops.cs

```diff
@@ -38,11 +38,11 @@ public void Loops_InstructionVisitedMaxTwice(string loop)
             Tag("End", arg);
             """;
         var validator = SETestContext.CreateCS(code, "int arg, int[] items", new AddConstraintOnInvocationCheck(), new PreserveTestCheck("arg")).Validator;
-        validator.ValidateExitReachCount(2);    // PreserveTestCheck is needed for this, otherwise, variables are thrown away by LVA when going to the Exit block
-        validator.TagValues("End").Should().HaveCount(2)
-            .And.SatisfyRespectively(
-                x => x.Should().HaveOnlyConstraints(ObjectConstraint.NotNull),
-                x => x.Should().HaveOnlyConstraints(ObjectConstraint.NotNull, TestConstraint.First));
+        validator.ValidateExitReachCount(3);    // PreserveTestCheck is needed for this, otherwise, variables are thrown away by LVA when going to the Exit block
```

Comment:
UT needs a rename

Reply:
On multiple places in this file

Reply:
I have adjusted the tests to make it easier to understand. Review per commit to see the exact changes.